target_chains/ethereum/contracts/Deploying.md

```diff
@@ -2,43 +2,25 @@
 
 ## EVM network configuration
 
-Each network that Pyth is deployed on has some configuration stored on this repo as described below:
-
-1. [`truffle-config.js`](./truffle-config.js) contains list of different networks with their configuration that includes:
-   - `provider`: The network provider is an HDWalletProvider with mnemonic stored in the $MNEMONIC
-     environment variable and the RPC URL of the network.
-   - `network_id`: Network ID of the chain.
-   - `gas`, `gasPrice` (optional): Usually RPCs estimate gas and gas price efficiently. Although some networks are not
-     good at it and you need to specify them manually.
-   - `timeoutBlocks` (optional): Number of blocks to wait for a transaction to make it to a block. If you specify a low
-     gas price the transaction will be sent but it will never get in a block. Sometimes the network gas price is volatile
-     (like Ethereum) and you need to wait longer than the default timeout.
-   - `networkCheckTimeout` (optional): RPC timeout for requests. Some RPCs might be very slow and you need to have this.
-   - `confirmations` (optional): Number of blocks to wait to consider the transaction final.
-   - `from` (optional): Public address of the mnemonic. Although it can be derived from `provider` some networks that
-     are not entirely EVM based need it.
-2. `.env.prod.<network>` contains the contract specific configuration for each network. It contains:
-   - `MIGRATIONS_DIR`: This is either [`./migrations/prod`](./migrations/prod) or
-     [`./migrations/prod-receiver`](./migrations/prod-receiver). The `prod-receiver` migrations should be used when you
-     need to deploy to a chain that is unsupported by the Wormhole network. The Wormhole Receiver contract acts as a
-     read-only Wormhole endpoint that can verify Wormhole messages even if the Wormhole network has not yet connected
-     the chain.
-   - `MIGRATIONS_NETWORK`: Network name in the [`truffle-config.js`](./truffle-config.js) file.
-   - `WORMHOLE_CHAIN_NAME`: Chain name in Wormhole. It is either defined in the
+Each network that Pyth is deployed on has some configuration stored on this repo in the contract manager package as described below:
+
+1. Chain configuration in [EvmChains.yaml](../../../contract_manager/store/chains/EvmChains.yaml) contains:
+   - `id`: id of the chain. This will be used for identifying which chain we want to deploy to.
+   - `wormholeChainName`: Chain name in Wormhole. It is either defined in the
      [Wormhole SDK constants](https://github.com/wormhole-foundation/wormhole/blob/dev.v2/sdk/js/src/utils/consts.ts)
      or is defined in [Wormhole Receiver names](../../../governance/xc_admin/packages/xc_admin_common/src/chains.ts). If the new
      network requires a Receiver contract you need to update the latter file and add the network there.
```

Comment:
I'm a bit afraid that people don't read this part or miss it. It's mostly a comment to my old self though 😅